### PR TITLE
Fixing the local server and config.json bug

### LINKED ISSUE #355: Config flask server is not working after removal of wrappers
**Is your feature request related to a problem? Please describe.**
After the shell wrappers were removed, the flask server doesn't create config.json file.

**Describe the solution you'd like**
To fix it.

**Additional context**
Yes

### PR DESCRIPTION
**Pull request by stealthanthrax**
Fixes #355 

Fixes the scripts after wrappers were removed.

### CODE REVIEW

**Comment by stealthanthrax** [comment]
@prateekiiest , have a look too

**Comment by stealthanthrax** [comment]
@hongquan , in this case it is necessary to change the directory as the scripts are being executed from systemd folder and if the directory wouldn't be changed, the JSON file is being created there.

File: access_point/server/server.py [hunk 1]
```diff
@@ -26,6 +36,7 @@ def config():
     tts = request.args.get('tts')
     hotword = request.args.get('hotword')
     wake = request.args.get('wake')
+    os.chdir(config_json_folder)
```
Comment (prateekiiest) [inline access_point/server/server.py]: use the `subprocess` functionality instead

**Comment by hongquan** [comment]
@stealthanthrax From systemd, you can set the working folder before running the script.

**Comment by stealthanthrax** [comment]
@hongquan , the `systemd` folder was just an example. Let us say I am in Documents folder, and to run the server I'll say `python3 /home/pi/SUSI.AI/susi_linux/access_point/server.py` and the hit endpoints, the config.json file will be created in the Documents folder itself

**Comment by hongquan** [comment]
@stealthanthrax Then it is the problem of the code, when it doesn't build the path for the to-be-produced config file.

**Comment by hongquan** [comment]
I will merge this PR as temporary fix to make SUSI working. Need to have a better fix later.

**Pull request merged**